controllers/kafkaacl_controller.go

```diff
@@ -57,11 +58,11 @@ func (h KafkaACLHandler) createOrUpdate(ctx context.Context, avn *aiven.Client,
 	}
 
 	// Creates it from scratch
-	r, err := avn.KafkaACLs.Create(
+	_, err = avnGen.ServiceKafkaAclAdd(
```

Comment:
The endpoint returns a list, and the old client looks for the id https://github.com/aiven/aiven-go-client/blob/main/kafka_acl.go#L31